openrr-apps/config/sample_teleop_config_urdf_viz_joy_gamepad.toml

```diff
@@ -0,0 +1,38 @@
+robot_config_path = "sample_robot_client_config_for_urdf_viz.toml"

```

Comment:
sample_teleop_config_urdf_viz_ros_joy_gamepad.toml might be better, but it's OK.